packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -802,7 +802,7 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
 
   bool get hasScrolledBody {
     for (final _NestedScrollPosition position in _innerPositions) {
-      assert(position.minScrollExtent != null && position.hasPixels);
+      assert(position.hasContentDimensions && position.hasPixels);
```

Comment:
Why would you like to make this change? Seems more or less equivalent to what we had before without benefits?

Reply:
Since the getter `minScrollExtent` is non-nullable. This would throw CastError
`Null check operator used on a null value`, when the underlying `_minScrollExtent` is null.

Reply:
Ah, I see. Makes sense!